static/app.ts

```diff
@@ -1,3 +1,15 @@
+class TopCtrl {
+  orderStatusService: OrderStatusService;
+
+  constructor(private OrderStatusService: OrderStatusService) {
```

Comment:
ditto here; if you want to give the member variable a different name, you can drop "private" from the argument list.